TRENZ.Docs.API/Models/Sources/AbstractFilesystemSource.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System.Text.RegularExpressions;
+using IOPath = System.IO.Path;
```

Comment:
Do we have a conflict?

Maybe `FSPath` for `FileSystemPath`?

Reply:
`AbstractFilesystemSource` has an auto-property named `Path`, yes.

I prefer `FileSystemPath`, actually. But it's quite long too...